src/mongocxx/test/bulk_write.cpp

```diff
@@ -257,19 +227,18 @@ TEST_CASE("passing write operations to append calls corresponding C function", "
         REQUIRE(called);
     }
 
-    SECTION("update_many invokes mongoc_bulk_operation_update_with_opts") {
-        auto bulk_update = libmongoc::bulk_operation_update_with_opts.create_instance();
-        update_func.many(true);
+    SECTION("update_many invokes mongoc_bulk_operation_update_many_with_opts") {
+        auto bulk_update = libmongoc::bulk_operation_update_many_with_opts.create_instance();
         bulk_update->visit(update_func);
 
         bw.append(model::update_many(filter, update_doc));
         REQUIRE(called);
     }
 
     SECTION(
-        "update_many with upsert invokes mongoc_bulk_operation_update_with_opts with upsert true") {
-        auto bulk_update = libmongoc::bulk_operation_update_with_opts.create_instance();
-        update_func.many(true);
+        "update_many with upsert invokes mongoc_bulk_operation_update_many_with_opts with upsert "
+        "true") {
```

Comment:
FWIW, this wrapping seems really odd.  Is that really our clang-format style? Ick.